resources/metadata/README.md

```diff
@@ -0,0 +1,22 @@
+# Shuttle Service Info
```

Comment:
Should this header also be renamed to Shuttle Metadata?

Reply:
Done!